mlflow/store/db/utils.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+
+def _upgrade_db(url):
+    """
+    Upgrade the schema of an MLflow tracking database to the latest supported version.
+    version. Note that schema migrations can be slow and are not guaranteed to be transactional -
+    we recommend taking a backup of your database before running migrations.
+
+    :param url Database URL, like sqlite:///<absolute-path-to-local-db-file>. See
+    https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls for a full list of valid
+    database URLs.
+    """
+    # alembic adds significant import time, so we import it lazily
+    from alembic import command
+    from alembic.config import Config
+
+    _logger.info("Updating database tables at %s", url)
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    package_dir = os.path.normpath(os.path.join(current_dir, os.pardir, os.pardir))
+    directory = os.path.join(package_dir, 'alembic')
+    config = Config(os.path.join(package_dir, 'alembic.ini'))
+    config.set_main_option('script_location', directory)
```

Comment:
Set migration script directory to `mlflow/alembic`, DB URL to the specified URL, and run command via the [Alembic commands API](https://alembic.sqlalchemy.org/en/latest/api/commands.html#commands) (this mimics what Airflow does). I QA'd that this works against a wheel built via `python setup.py bdist_wheel`, just to check that the relative paths and such work out within a wheel installation as well.